evaluation/evaluator.go

```diff
@@ -318,8 +318,7 @@ func (e Evaluator) evaluate(identifier string, target *Target, kind string) (res
 	if err != nil {
 		return rest.Variation{}, err
 	}
-
-	if flag.Kind != kind {
+	if string(flag.Kind) != kind {
```

Comment:
ooc does `Kind` have a `.String()` method on it?

Reply:
It does not